### ISSUE TITLE
Feature Request: hide window content when dragging

### ISSUE THREAD

**Issue by MingcongBai** [mate-desktop/marco#291]
Currently with Marco 1.16.0 (GTK+ 3 build), there doesn't seem to be a way to hide window content while dragging the windows - which resulted in a huge lag on my iBook G4, which comes with an ATI Radeon 9550.

I had similar issue with Xfce (xfwm4), but after disabling window content display while dragging windows, the desktop appeared to be way more responsive.

Request: a setting to hide window content when dragging?

**Comment by monsta** [comment]
There is a "reduced resources" option. It includes hiding window content when dragging the window. A wireframe will be shown instead.

> If true, marco will give the user less feedback by using wireframes, avoiding animations, or other means.  This is a significant reduction in usability for many users, but may allow legacy applications to continue working, and may also be a useful tradeoff for terminal servers.  However, the wireframe feature is disabled when accessibility is on.

You can enable it via dconf-editor (```org > mate > marco > general > reduced-resources```).

**Comment by MingcongBai** [comment]
Could this ever be made into a graphical setting?

2016年11月14日 14:00，"monsta" <notifications@github.com>写道：

**Comment by monsta** [comment]
Maybe... not sure when though.

**Comment by kn00tcn** [comment]
i think i should pick up this simple gui toggle to learn how to contribute & have important people not waste time

**Comment by flexiondotorg** [comment]
MATE Tweak, included in Ubuntu MATE and Debian, includes an toggle for this
setting.

Regards, Martin.

**Comment by kn00tcn** [comment]
there is no mate tweak in fedora & probably others, if there are actual options available in dconf, they should also be available in gui

**Comment by flexiondotorg** [comment]
I don't agree that just because an option exists it has to be presented in
the UI. Many of the options not exposed via the UI are there for distro
maintainers and system integrators to configure via gsettings overrides.

But MATE Tweak does provide the functionality you're seeking.

Regards, Martin.

**Comment by raveit65** [comment]
> there is no mate tweak in fedora & probably others, if there are actual options available in dconf, they should also be available in gui

Of course 95% of settings in mate-tweak are gsettings keys and dconf-editor is the GUI.
Unfortunately developer dropped the sidebar in dconf-editor for gtk+-3.22 (f25), which make this app more unusable if you don't know where are the keys are.
[rave@mother ~]$ gsettings get org.mate.Marco.general reduced-resources
false
.......is the key for it.

PS: i do not add any application to fedora/epel7 without an active co-maintainer.

**Comment by kn00tcn** [comment]
martin, i was thinking about it, maybe 'all' is too strong of a word, but it's the user related ones that shouldnt be hidden

this thread is a perfect example, the user would logically think to open the window settings to disable some 'demanding' effect, it's not distro specific or like a workaround for some compositor bug

another example i spotted from april, couple guys needed to speed up the panel auto hide for productivity purposes, certainly a logical use case to have in the gui, they thought there was no option for it, in fact it's not even there in dconf (on fedora at least)

i happened to casually be reading arch wiki when i noticed it mentioned the hidden animation speed+delay settings that you have to manually add via terminal, so i was able to answer the issue thread, but unfortunately almost a year late...

for dconf settings, i would at least have expected all available ones to be visible with their defaults so someone poking around might try tweaking for themselves if they felt like it, but it sure is weird that panel hide animation & delay are completely missing unless you knew how to create them by name
